lib/xnvme_be_spdk_dev.c

```diff
@@ -572,10 +572,7 @@ enumerate_attach_cb(void *cb_ctx, const struct spdk_nvme_transport_id *trid,
 		}
 		ns = spdk_nvme_ctrlr_get_ns(ctrlr, nsid);
 		if (!ns) {
-			continue;
-		}
-		if (!spdk_nvme_ns_is_active(ns)) {
-			XNVME_DEBUG("FAILED: skipping inactive nsid: %d", nsid);
+			XNVME_DEBUG("FAILED: spdk_nvme_ctrlr_get_ns()");
```

Comment:
How about adding the ``nsid`` to the debug output here? Might be useful?

Reply:
Good catch, I agree